MobileCoin.podspec

```diff
@@ -58,7 +58,7 @@ Pod::Spec.new do |s|
       "Sources/Network/*.{h,m,swift}",
     ]
 
-    subspec.dependency "LibMobileCoin/Core", "~> 1.2.0-pre7"
```

Comment:
@carybakker 

This is why your CI tests were failing. the `~> `1.2.0-pre7` means choose a version equal to or greater than specified, but within the same major version (excludes >= 2.x.x).

I changed to comma syntax which is same as "version equals".